draft-ietf-httpbis-digest-headers.md

```diff
@@ -284,6 +282,9 @@ A sender MAY send a representation-data-digest using a digest-algorithm without
 knowing whether the recipient supports the digest-algorithm, or even knowing
 that the recipient will ignore it.
 
+Digest MAY be cached and the freshness information of the resource, eventually conveyed by validator
```

Comment:
Generally, we try to avoid re-stating requirements in other specs, or constraining how they're interpreted, because that makes the composition of the specs more brittle -- there's more chance of misinterpretation or unintentional misalignment (either as written or after some future change in the dependency).

So if you say anything at all, it shouldn't be a RFC2119 requirement; just some prose about the relationship for reader convenience.

Reply:
Thanks @mnot. Let's drop those lines then!